docs/src/opensourcetpa.md

```diff
@@ -0,0 +1,294 @@
+# Open source TPA
+
+Trusted Postgres Architect has historically been built to address the deployment needs of EDB customers. Among its playbooks is the ability to deploy EDB-recommended architectures using EDB repositories containing EDB versions of Postgres and other tools.
```

Comment:
playbook is an Ansible-specific thing and there are no playbooks in TPA

Reply:
Will edit that out.

Reply:
Reworked